### ISSUE TITLE
ngAria - incorrect "aria-checked" and "tabindex" applied to checkbox/radio

### ISSUE THREAD

**Issue by DmitryEfimenko** [angular/angular.js#10212]
Hello angular team!

**Overview of the Issue** - incorrect attributes `aria-checked` and `tabindex` are applied to the input of type `radio`: when radio is selected, the following attributes are applied: `aria-checked="false"`, `tabindex="-1"`.
**Motivation for or Use Case** - radio button never gets selected when pressing *Tab*
**Angular Version(s)** - 1.3.1 for all modules
**Browsers and Operating System** - Chrome 38.0.2125.111 m. Haven't tested in other browsers, but pretty sure it'll be the same.
**Reproduce the Error** - see [Plunker](http://plnkr.co/edit/liOsgkByxFgPeqF7iuyc?p=preview).
**Related Issues** - not found
**Suggest a Fix** - Switching attribute `value` to `ng-value` fixes the issue, but it is inconsistent through out the angular's logic - it was able to figure out that the radio should be selected with the regular `value` attribute, but it could not figure out correct accessibility attributes. It appears the issue has to do with the following [lines of code](https://github.com/angular/angular.js/blob/5b23bc9b07f8a4363c9c549a248f690b1a31b7be/src/ngAria/aria.js#L232-L233)
